plugins/cache_promote/cache_promote.cc

```diff
@@ -333,6 +429,66 @@ class LRUPolicy : public PromotionPolicy
     return "LRU";
   }
 
+  bool
+  stats_add(const char *remap_id) override
```

Comment:
This all looks weird to me. Why not break out the code inside of the for() loop into a function (create_stat() or whatever, and then just have 8 calls like this:

```
cache_hits  = create_stat("cache_hits");
freelist_size_id = create_stat("freelist_size");
...
```

Reply:
Okay, I made the change.